### DISCUSSION TITLE
How to get index file size by code?

### DISCUSSION THREAD

**Discussion by Alison-starbeat** [milvus-io/milvus#22327] [Q&A and General discussion]
Sorry for disturbing all experts in Milvus……
I'm new at Milvus, and the version of Milvus and pyMilvus is 2.0.2. 
I hope to choose more suitable index for my data, so I should compare **the real sizes** of different indexes, like IVF_SQ8,HNSW, IVF_PQ and so on.

After searching all docs of Milvus, I still couldn't find **how to get index file size(or index file path)**……
P.S. Once I find some methods provided by class Milvus, but then I find _this method(Milvus) will be deprecated_, it suggests me finding new methods in _Collection/utility_,and I don't find any useful information in these methods.

And I could not use some tools like  Attu.
If possible, could someone tell me how to do it? Thanks for your help!

**Comment by xiaofan-luan** [comment]
Hi @Alison-starbeat , first of all you should use V2.2.3, this will save you a lot of trouble, trust me.

https://milvus.io/tools/sizing is the tool you are looking, it gives you some idea about memory usage index costing.

**Reply by Alison-starbeat** [reply]
Thank you for your suggestion! But as I have finished lots of the project by using the outdated version of Milvus, there is no turning back for me now TAT……
Anyway, your suggestions help me a lot!
